### PR TITLE
Update writing-a-blog-post.md

### PR DESCRIPTION
**Pull request by justinegeffen**
General editing and added editorial guidelines around structuring posts and content ideas.

### CODE REVIEW

**Comment by justinegeffen** [comment]
This started off as more of a step-by-step guide (https://docs.google.com/document/d/13GO3bg1BwBm4skkf4Kat1JcLLOP1c_OJt6OA1MD5eW8/edit#) but morphed into more of a blog post itself. Open to changing it back if it works better!

File: site/content/internal/writing-a-blog-post.md [hunk 1]
```diff
@@ -5,11 +5,56 @@ section: internal
...
-Went to a conference recently? Worked on something cool? Got something else Mattermost-related you want to post about? Follow these steps to write a blog post:
+Been to a conference recently? Worked on something cool? Got something else Mattermost-related you want to post about? Writing a blog post is a great way to share your experience with the community. 
 
-1. Clone https://github.com/mattermost/mattermost-developer-documentation 
-2. Create a new .md file in the [/site/content/blog/](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/blog) folder
-  - Use `YYYY-MM-DD-<your-blog-post-title>.md` as the filename
+Blog posts can cover a wide range of topics, such as:
+
+- Addressing a customer-facing problem
+- Describing an experience with Mattermost/your Mattermost implementation
```
Comment (amyblais) [inline site/content/internal/writing-a-blog-post.md]:
```suggestion
- Describing an experience with Mattermost / your Mattermost implementation 
```

File: site/content/internal/writing-a-blog-post.md [hunk 2]
```diff
@@ -5,11 +5,56 @@ section: internal
...
-  - Use `YYYY-MM-DD-<your-blog-post-title>.md` as the filename
+Blog posts can cover a wide range of topics, such as:
+
+- Addressing a customer-facing problem
+- Describing an experience with Mattermost/your Mattermost implementation 
+- Sharing information about cool tech 
+- Sharing feedback on an interesting talk or conference 
+- Part of a Hackathon project
+- A Help Wanted ticket
+- A knowledge-share and call for feedback/community engagement
```
Comment (amyblais) [inline site/content/internal/writing-a-blog-post.md]:
```suggestion
- A knowledge-share and call for feedback / community engagement
```

File: site/content/internal/writing-a-blog-post.md [hunk 3]
```diff
@@ -5,11 +5,56 @@ section: internal
...
+- A discussion of a specific problem or improvement that you worked on
+- A breakdown of a new process or technology you’re using
+
+
+Once you've got the topic in mind - what it's about, what you want to achieve with the post, what the next steps are etc. it’s sometimes helpful to start writing the conclusion and expand to your jumping off point to introduce your topic/idea/discovery draws
```
Comment (amyblais) [inline site/content/internal/writing-a-blog-post.md]:
```suggestion
Once you've got the topic in mind - what it's about, what you want to achieve with the post, and what the next steps are - it’s sometimes helpful to start writing the conclusion and expand to your jumping off point to introduce your topic/idea/discovery draws.
```

File: site/content/internal/writing-a-blog-post.md [hunk 4]
```diff
@@ -5,11 +5,56 @@ section: internal
...
+
+Make a note of your intended audience, so you can decide whether to use very technical terms/jargon, or spend time unpacking terminology.
+
+
+Structuring Your Blog Post
+--------------------------
+
+These are some ideas of the parts of a blog post. They’re not mandatory and not all blog posts will include every aspect. What works for some posts won’t work for others. 
+
+- **Introduction/overview:** An opening paragraph detailing the goal of the post and the technologies/processes that were used. For example: “Monitoring is an essential part of our organization. When we started, we were using “x” which gave us insight into “y”. With our growth as a company, we need more insights into areas that “x” can’t handle, so we decided to switch to “a” and “b””.
```
Comment (amyblais) [inline site/content/internal/writing-a-blog-post.md]:
```suggestion
- **Introduction/Overview:** An opening paragraph detailing the goal of the post and the technologies/processes that were used. For example: “Monitoring is an essential part of our organization. When we started, we were using “x” which gave us insight into “y”. With our growth as a company, we need more insights into areas that “x” can’t handle, so we decided to switch to “a” and “b””.  
```

File: site/content/internal/writing-a-blog-post.md [hunk 5]
```diff
@@ -5,11 +5,56 @@ section: internal
...
+Make a note of your intended audience, so you can decide whether to use very technical terms/jargon, or spend time unpacking terminology.
+
+
+Structuring Your Blog Post
+--------------------------
+
+These are some ideas of the parts of a blog post. They’re not mandatory and not all blog posts will include every aspect. What works for some posts won’t work for others. 
+
+- **Introduction/overview:** An opening paragraph detailing the goal of the post and the technologies/processes that were used. For example: “Monitoring is an essential part of our organization. When we started, we were using “x” which gave us insight into “y”. With our growth as a company, we need more insights into areas that “x” can’t handle, so we decided to switch to “a” and “b””.  
+- **Problem/Solution/Situation:** Detail the problem or scenario that sparked the blog post. For example, “our monitoring tools weren’t giving us the insight we needed, and were costing a lot of money. We decided to try solve this by using multiple tools that could be used or on standby as needed. This saves money.”
```
Comment (amyblais) [inline site/content/internal/writing-a-blog-post.md]:
```suggestion
- **Problem/Solution/Situation:** Detail the problem or scenario that sparked the blog post. For example, “Our monitoring tools weren’t giving us the insight we needed, and were costing a lot of money. We decided to try solve this by using multiple tools that could be used or on standby as needed. This saves money.” 
```

File: site/content/internal/writing-a-blog-post.md [hunk 6]
```diff
@@ -5,11 +5,56 @@ section: internal
...
+
+- **Introduction/overview:** An opening paragraph detailing the goal of the post and the technologies/processes that were used. For example: “Monitoring is an essential part of our organization. When we started, we were using “x” which gave us insight into “y”. With our growth as a company, we need more insights into areas that “x” can’t handle, so we decided to switch to “a” and “b””.  
+- **Problem/Solution/Situation:** Detail the problem or scenario that sparked the blog post. For example, “our monitoring tools weren’t giving us the insight we needed, and were costing a lot of money. We decided to try solve this by using multiple tools that could be used or on standby as needed. This saves money.” 
+- **Environment:** Plugins, software, specific configurations/services used. 
+- **Steps:** The process followed to get from conception to implementation, for example, how the monitoring was configured initially, and the new configuration. Include code samples/screenshots. How long it took to build a database/history. What steps were taken to create a good alert system.
```
Comment (amyblais) [inline site/content/internal/writing-a-blog-post.md]:
```suggestion
- **Steps:** The process followed to get from conception to implementation, for example, how the monitoring was configured initially, and the new configuration. Include code samples and/or screenshots. How long it took to build a database/history. What steps were taken to create a good alert system. 
```

File: site/content/internal/writing-a-blog-post.md [hunk 7]
```diff
@@ -5,11 +5,56 @@ section: internal
...
+- **Problem/Solution/Situation:** Detail the problem or scenario that sparked the blog post. For example, “our monitoring tools weren’t giving us the insight we needed, and were costing a lot of money. We decided to try solve this by using multiple tools that could be used or on standby as needed. This saves money.” 
+- **Environment:** Plugins, software, specific configurations/services used. 
+- **Steps:** The process followed to get from conception to implementation, for example, how the monitoring was configured initially, and the new configuration. Include code samples/screenshots. How long it took to build a database/history. What steps were taken to create a good alert system. 
+- **Samples:** Code samples are often useful, as are screenshots/animated GIFs if a complex process is being demonstrated. 
+- **Benefits:** The benefits of the exercise or process of the blog post - man-hours saved, budget target achieved, lower overheads etc.
```
Comment (amyblais) [inline site/content/internal/writing-a-blog-post.md]:
```suggestion
- **Benefits:** The benefits of the exercise or process of the blog post - man-hours saved, budget target achieved, lower overheads, etc. 
```

File: site/content/internal/writing-a-blog-post.md [hunk 8]
```diff
@@ -5,11 +5,56 @@ section: internal
...
+- **Problem/Solution/Situation:** Detail the problem or scenario that sparked the blog post. For example, “our monitoring tools weren’t giving us the insight we needed, and were costing a lot of money. We decided to try solve this by using multiple tools that could be used or on standby as needed. This saves money.” 
+- **Environment:** Plugins, software, specific configurations/services used. 
+- **Steps:** The process followed to get from conception to implementation, for example, how the monitoring was configured initially, and the new configuration. Include code samples/screenshots. How long it took to build a database/history. What steps were taken to create a good alert system. 
+- **Samples:** Code samples are often useful, as are screenshots/animated GIFs if a complex process is being demonstrated. 
+- **Benefits:** The benefits of the exercise or process of the blog post - man-hours saved, budget target achieved, lower overheads etc. 
+- **Conclusion:** Whether the exercise or process has long-term potential, whether it’s still in place, or whether it was a failure. 
+
+Some popular blogs that are worth reading include:
```
Comment (amyblais) [inline site/content/internal/writing-a-blog-post.md]: These are not rendering as a list

File: site/content/internal/writing-a-blog-post.md [hunk 9]
```diff
@@ -32,8 +77,6 @@ Went to a conference recently? Worked on something cool? Got something else Matt
     <some more content>
     ```
 
-4. Write your blog post!
-5. Submit a pull request to https://github.com/mattermost/mattermost-developer-documentation and assign two dev reviews and an editor review from @amyblais
-6. Once merged it should show up on [developers.mattermost.com/blog](https://developers.mattermost.com/blog) within 10-15 minutes. When it shows up, post about it in the Developers channel on community.mattermost.com!
-
-Looking for examples? The [/site/content/blog/](https://github.com/mattermost/mattermost-developer-documentation/tree/master/site/content/blog) folder has a bunch!
+4. Write your blog post. 
+5. Submit a pull request to https://github.com/mattermost/mattermost-developer-documentation and assign two dev reviews and an editor review from @amyblais.
```
Comment (amyblais) [inline site/content/internal/writing-a-blog-post.md]:
```suggestion
5. Submit a pull request to https://github.com/mattermost/mattermost-developer-documentation and assign two dev reviews and an editor review from @amyblais or @justinegeffen.
```

File: site/content/internal/writing-a-blog-post.md (hunk 1 above)
Comment (justinegeffen) [reply site/content/internal/writing-a-blog-post.md]: I generally don't use a space when using a slash but I know in our documentation we have instances where we do and instances where we don't. Grammatically it's meant to be the latter but happy to agree on one, standardize it, and then apply across the board. Thoughts?

File: site/content/internal/writing-a-blog-post.md (hunk 2 above)
Comment (justinegeffen) [reply site/content/internal/writing-a-blog-post.md]: See comment above.

File: site/content/internal/writing-a-blog-post.md (hunk 8 above)
Comment (justinegeffen) [reply site/content/internal/writing-a-blog-post.md]: Ah I didn't check that - I'll fix the formatting!

File: site/content/internal/writing-a-blog-post.md (hunk 1 above)
Comment (amyblais) [reply site/content/internal/writing-a-blog-post.md]: We can leave as is for now here.

**Pull request merged**